transformations/country_state_abbreviation_transformation/transformation.py

```diff
@@ -0,0 +1,112 @@
+import io
+import json
+import os
+import random
+import re
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class CountryStateAbbreviation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+    ]
+    languages = ["en"]
+
+    def __init__(self, seed=0, country=True, state=True, country_filter='USA', abbr=True, exp=True):
+        super().__init__(seed)
+        self.country = country
+        self.state = state
+        self.country_filter = country_filter
+        self.abbr = abbr
+        self.exp = exp
+
+    def generate(self, sentence: str):
+        perturbed_texts = self.country_state_abbreviation(text=sentence)
+        return perturbed_texts
+
+    def build_mapping(self):
+        __location__ = os.path.realpath(
+            os.path.join(os.getcwd(), os.path.dirname(__file__))
```

Comment:
Can you make this not depend on the cwd? This could lead to errors when scripts are called from different directories.